source/common/http/conn_manager_impl.cc

```diff
@@ -1594,6 +1598,8 @@ void ConnectionManagerImpl::ActiveStream::encodeHeaders(ResponseHeaderMap& heade
     state_.is_tunneling_ = true;
   }
 
+  blockRouteCache();
```

Comment:
Please add a comment explaining why it is ok to block the route-cache at this stage.

Reply:
get it.